src/controllers/hooks.js

```diff
@@ -266,6 +266,23 @@ function buildSlackMessage(baseURL, source, hlxConfig, channel) {
 function HooksController(lambdaContext) {
   const { dataAccess } = lambdaContext;
 
+  // todo: temporary to back fill hlx config for existing sites
+  /* c8 ignore start */
```

Comment:
please do not introduce TODOs and `c8 ignore`s in a mature part of our codebase.

Reply:
i will leave the TODO as this code is temporary and needs to be removed after a while. will add test case.